.github/CODEOWNERS

```diff
@@ -1,5 +1,27 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
+#
+# What is a "CODEOWNER"?
+#
+# A CODEOWNER MUST ____ for packages they own:
+# - introduce themselves at a Python SIG meeting
+# - demonstrate advanced knowledge of the dependencies
+# - respond to issues when they are notified
+# - fix failing unit tests or any other blockers to the CI/CD workflow
+# - update usage of `opentelemetry-python-core` APIs upon the introduction of breaking changes
+# - respond within 2~3 business days for any of the above asks
```

Comment:
I think this is a bit too much of a hard requirement. Some times we don't even respond withing 2-3 days for our core components either. Also adding many requirements here may scare people away from wanting to be owners :sweat_smile:

Reply:
Sounds good, I removed this as a requirement!